activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+*   Add attribute encryption support.
+
+    Encrypted attributes are declared at the model level. These
+    are regular active record attributes backed by a column with
+    the same name. The system will transparently encrypt these
+    attributes before saving them into the database and will
+    decrypt them when retrieving their values.
+    
+
+    ```ruby
+    class Person < ApplicationRecord
+      encrypts :name
+      encrypts :email_address, deterministic: true
+    end
+    ```
+
+    You can learn more in the [Active Record Encryption
```

Comment:
Maybe we can avoid documentation link. It would be anyways accessible with simple search. 🤔

Reply:
I think it's useful if someone is reading the CHANGELOG but I can definitely remove if it's against convention